src/pager.c

```diff
@@ -4396,6 +4396,9 @@ static int syncJournal(Pager *pPager, int newHdr){
 */
 static int pager_write_pagelist(Pager *pPager, PgHdr *pList){
   int rc = SQLITE_OK;                  /* Return code */
+  int numWriteReqs = 0;
+  int requestsAlloc = 100;
+  WriteRequest **requests = (WriteRequest**)malloc(requestsAlloc*sizeof(WriteRequest*));
```

Comment:
Modified `pager_write_pagelist` to accumulate contiguous write requests, as well as issue concurrent write requests.